src/app/components/search/SearchResultsCards/ItemCard.module.css

```diff
@@ -56,6 +55,7 @@
     display: flex;
     flex: 0 0 42px;
     flex-direction: row;
+    margin: 0 -8px 0 0;
```

Comment:
just reducing the gap between the checkbox and content, not necessary since the checkbox has a lot of padding already